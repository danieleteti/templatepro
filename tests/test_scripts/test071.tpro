{{:dataitemnullable.NullString}}
{{:dataitemnullable.NullBoolean}}
{{:dataitemnullable.NullInt32}}
{{:dataitemnullable.NullInt64}}
{{:dataitemnullable.NullDate}}
{{:dataitemnullable.NullDateTime$}}
{{:dataitemnullable.NullTime}}
{{:dataitemnullable.NullCurrency}}