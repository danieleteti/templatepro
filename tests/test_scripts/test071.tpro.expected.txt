Daniele
True
123
234
1979-11-04
1979-11-04T17:18:19.000+01:00
17:18:19
1234.5678